appeals/api/src/server/endpoints/appeals/appeals.formatter.js

```diff
@@ -89,6 +90,7 @@ const formatAppeal = (appeal, folders) => {
 			decision: {
 				folderId: folders[0].id,
 				outcome: appeal.inspectorDecision?.outcome,
+				// @ts-ignore
```

Comment:
The error this is ignoring is also present on main, so it wasn't introduced in this PR (not sure how it got past tscheck)